lib/scooter/httpdispatchers/orchestrator/v1/v1.rb

```diff
@@ -89,6 +89,21 @@ def post_inventory(payload)
         def get_status
           @connection.get("#{@version}/status")
         end
+
+        #task endpoints
+        def get_task(mod, task)
+          @connection.get("#{@version}/tasks/#{mod}/#{task}")
+        end
+
+        def get_tasks
+          @connection.get("#{@version}/tasks")
+        end
+
+        def run_task(payload)
```

Comment:
My only reason for commenting is that is the naming convention that scooter has used.  I think that `run_tasks` makes more sense, but this file is just intended to represent actions on endpoints for better or worse.